examples/ex1.cpp

```diff
@@ -88,6 +88,13 @@ int main(int argc, char *argv[])
    }
    args.PrintOptions(cout);
 
+   // FIXME
```

Comment:
What needs fixing here?

Reply:
The numbering and moving `Enable()` inside `Configure()`.

Reply:
Instead of
```cpp
Device::Configure(device);
...
Device::Disable();
```
wouldn't it be easier to understand

```cpp
Device::Enable(device);
...
Device::Disable();
```
?

Reply:
There is no longer need for `Device::Disable()`, we only need to configure/enable/setup the device in the beginning of the example.